static/js-functions.js

```diff
@@ -38,10 +38,12 @@ $('#test').on("click", ()=>alert("hello"));
 // #audio reffering to the id 
 // .class reffere to the class of elements wit the 
 
-
+// You have to get the word from where you placed in the HTML. For example, suppose the word is on 
+// an element with an id named 'word'
+var farsi_word = document.getElementById('word')
 
 function getPronouciation(){
```

Comment:
You can also pass the word as an argument to this function if you want. Just remember that the problem we're trying to solve with this is to send the word in the url for your request. You cannot use `<farsi_word>` It has to be the actual word you're requesting.